lib/filterx/filterx-expr.c

```diff
@@ -76,6 +77,28 @@ filterx_expr_format_location_tag(FilterXExpr *self)
     return evt_tag_str("expr", "n/a");
 }
 
+GString *
+filterx_expr_format_location(FilterXExpr *self)
+{
+  GString *location = scratch_buffers_alloc();
+
+  if (self && self->lloc)
+    g_string_printf(location, "%s:%d:%d", self->lloc->name, self->lloc->first_line, self->lloc->first_column);
+  else
+    g_string_assign(location, "n/a");
+
+  return location;
+}
+
+const gchar *
+filterx_expr_get_text(FilterXExpr *self)
+{
+  if (self && self->expr_text)
```

Comment:
right now expr_text is only available if debug is enabled. we might want to make that available even on the default log level.

Reply:
Yes, I'll enable it.